templates/todo/api/js/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es5",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "es2022",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```

Comment:
This is too ambitious. Node 16 (which is current LTS) does not support all Ecmascript 2022 features: https://node.green/. 

Your safe+modern bet is `es2020`